tools/accuracy_checker/accuracy_checker/adapters/segmentation.py

```diff
@@ -104,12 +104,33 @@ class BrainTumorSegmentationAdapter(Adapter):
     __provider__ = 'brain_tumor_segmentation'
     prediction_types = (BrainTumorSegmentationPrediction, )
 
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'make_argmax': BoolField(
+                optional=True, default=False, description="Allows to apply argmax operation to output values."
+            ),
+            'label_order': ListField(
+                optional=True, default=[1, 2, 3], description="Specifies order of output labels, according to "
```

Comment:
please specify value_type for avoid getting unexpected values

Reply:
Ok, thanks!